README.md

```diff
@@ -128,34 +130,70 @@ The optional boolean argument "includeAuthorizationData" specifies
         scopes are not returned.
 
 
-An "authResponse" object have sevral value accesible with this library:
+An `authResponse` object have sevral value accesible with this library:
 
 #### id_token
 
-Accessible with "Gapi.id_token", this "string" value represent the ID
+Accessible with `Gapi.id_token`, this "string" value represent the ID
 token granted.
 
 [...TODO add other authResponse values...]
 
 
-## How to use a "googleAuth" result
+## How to use a `googleAuth` result
 
-Like said above, the function "Gapi.auth2_init" return a "googleAuth",
+Like said above, the function `Gapi.auth2_init` return a `googleAuth`,
 that type can be used with other function in order to use function that
 are linked to the object.
 
-### Gapi.then_
+### `Gapi.then_`
 
-This is the main use of a "googleAuth", this function take 2 "callbacks"
+This is the main use of a `googleAuth`, this function take 2 "callbacks"
 in parameters: "onInit" and "onError". It returns a promise that will
-calls the "onInit" function when the "GoogleAuth" object is fully
+calls the "onInit" function when the `GoogleAuth` object is fully
 initialized or the "onError" function if an error is raised while
 initializing.
 
 Example:
 
-```Shell
+```Ocaml
 Gapi.then_ init
            ~onInit:(fun auth -> wakeup (Ok auth))
            ~onError:(fun msg -> wakeup (Error msg))
 ```
+
+## Module `Lwt`
+
+This module `Lwt` provide alternatives implementations of `Gapi`
+functions that return a "promise" of a result, under the forme of "_
+Lwt.t" objects.
+
+[Learn more about Lwt](https://ocsigen.org/lwt/latest/manual/manual)
+
+### `Gapi.Lwt.load`
+This version of the `load` function can't receive a "callback" argument,
+because it is used in order to "waikeup" a thread in a "wait" state.
```

Comment:
```suggestion
This version of the `load` function can't receive a *callback* argument,
because it is used in order to *wakeup* a thread in a *wait* state.
```